{"uuid": "2d3df01a-3579-40ce-8221-1054c7412b69", "children": ["f2b0a9ca-25f8-4751-8706-f045677dde5b"], "befores": [{"name": "create_token", "status": "broken", "statusDetails": {"message": "TypeError: verfiy_http_status_code() got an unexpected keyword argument 'expected_data'\n", "trace": "  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1132, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/jabir/APIAutomation/.venv/lib/python3.10/site-packages/_pytest/fixtures.py\", line 913, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"/home/jabir/APIAutomation/tests/test/crud/test_create_full_booking.py\", line 32, in create_token\n    verfiy_http_status_code(response_data=response, expected_data= 200)\n"}, "start": 1712376126316, "stop": 1712376127460}], "start": 1712376126316, "stop": 1712376127518}